pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -1299,18 +1458,25 @@ def run_restore_from_backup_cmd(self, backup_path):
         )
 
     def reset_near_home(self):
+        # Chunk validators don't have initial state and thus a backup.
+        if self.config.get('role') == 'validator':
+            self.remove_data_dir()
+            self.set_state(TestState.STOPPED)
+            return
+
         backup_id = self.data['state_data']
         if backup_id is None:
             backup_id = 'start'
```

Comment:
Chunk validators do not have the inital backup. Subsequent backups can still be created and we should support restoring from them.
```suggestion
        backup_id = self.data['state_data']
        if backup_id is None:
            backup_id = 'start'
         # Chunk validators don't have initial state and thus a backup.
        if backup_id == 'start' and self.config.get('role') == 'validator':
            self.remove_data_dir()
            self.set_state(TestState.STOPPED)
            return
```